scripts/apps/search/directives/DateFilters.ts

```diff
@@ -7,6 +7,24 @@ interface IDateRange {
     };
 }
 
+const nextMonthFilter: IDateRange = {
+    key: 'next_month',
+    label: gettext('Next Month'),
+    elasticSearchDateRange: {
+        lte: 'now+1M/M',
+        gte: 'now+1M/M',
+    },
+};
+
+const next3MonthsFilter: IDateRange = {
+    key: 'next_3_months',
+    label: gettext('Next 3 Months'),
+    elasticSearchDateRange: {
+        lte: 'now+3M/M',
+        gte: 'now+3M/M',
+    },
```

Comment:
If today is 2nd Oct, should next 3 months mean 2nd October to 31st December inclusive or November 1st - January 1st inclusive or something different?

Reply:
it should be from today (Oct 2nd) to January 2nd inclusive

Reply:
I think you should round it to day then `/d` otherwise it would be till the end of month `/M`

Reply:
You're right. I'm also thinking I need to remove `gte` as we also need old articles to be marked as "overdue"